css/scss/layout/_main.scss

```diff
@@ -0,0 +1,63 @@
+main {
+  padding: 30px;
+  text-align: center;
+
+  button {
+    @include button_effect($red);
+  }
+
+  .queue {
+    display: grid;
+    grid-gap: 50px;
+    grid-template-columns: repeat(auto-fill, minmax(200px, auto));
+    margin: 50px 0;
+    text-align: left;
+    counter-reset: pizza;
+  }
+}
+
+.pizza {
+  position: relative;
+  background-color: $dark-red;
+  border: 5px solid $dark-gray;
+  counter-increment: pizza;
+  transition: .3s;
+
+  img {
+    max-width: 100%;
+  }
+
+  &::before {
+    content: counter(pizza);
```

Comment:
I dont think that its a good case for counter, because its not order number, its ID of pizza. Also pizza should be sorted by time to resolve. So ID !== order.